invenio_communities/assets/semantic-ui/js/invenio_communities/subcommunity/SubcommunityResults.js

```diff
@@ -0,0 +1,75 @@
+// This file is part of Invenio
+// Copyright (C) 2024 CERN.
+//
+// Invenio is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import { InvenioSearchPagination } from "@js/invenio_search_ui/components";
+import { i18next } from "@translations/invenio_requests/i18next";
+import PropTypes from "prop-types";
+import React from "react";
+import { Count, ResultsList, Sort } from "react-searchkit";
+import { Grid, Segment } from "semantic-ui-react";
+
+export const SubcommunityResults = ({
```

Comment:
this exact code appears also in invenio_requests. Why do we need a custom component for this?

Reply:
yep, that was my [comment](https://github.com/inveniosoftware/invenio-communities/pull/1242#discussion_r1813847641) . Let's follow up there